Ros2 update (#36)
husarion/panther-docker: README.md

> Latest Panther Docker images are compatible with Built-in Computer OS version 2.0.0 and newer. If your operating system is older, please ensure you update it before proceeding. **[COMMING SOON]** Follow [operating system reinstallation](ros2-os-instalation-link) for more info. 
 
> [!Note] Latest Panther Docker images are compatible with Built-in Computer OS version 2.0.0 and newer. If your operating system is older, please ensure you update it before proceeding. **[COMING SOON]** Follow [operating system reinstallation](ros2-os-instalation-link) for more info.

Connect to Panther's Built-in Computer:

```bash
ssh husarion@10.15.20.2
```

Edit Docker compose file:
```bash
nano compose.yaml
```
## Quick Start

Update Docker image tag:
```yaml
  panther_ros:
    image: husarion/panther:<newest-stable-tag> # example tag: humble-2.0.0-20231208-stable
```
Depending on your needs, you can run Docker to quickly launch the physical robot or run a simulation. To do this, clone this repository to your robot or computer.

Restart Docker containers:
```bash
docker compose up -d --force-recreate
git clone -b ros2 https://github.com/husarion/panther-docker.git
cd panther-docker/demo
```

> ## Running Simulation

Author: I'd belive we want to user have easy acces to the leatest stable wersion. Thats why we have tag `husarion\panther-$ROS-DISTRO` which point to the latest stable version, if version is not stable, the image always contains varsion and date after $ROS_DISTRO.
Thank's to this the only command which user must execute is `docker compose pull`

> ## Running Simulation

Reviewer: Two things:
- compose.yaml file will be available on Panthers Rpi, no point in cloning repo there - just modify tag
- for all ours docker images marked as stable tags should have dates, and this is true for all robots. There is no universal tag that points to stable version. So to use new stable version it is necessary to modify compose file

> ## Running Simulation

Author: It is universal tag `humble`

### 💻 Gazebo Simulation

To give Docker access to your screen run:

```bash
xhost local:docker
docker compose -f compose.simulation.yaml up
```

> [!NOTE]

> > If you have an NVIDIA GPU, it is worth changing the compos configuration from cpu-config to gpu-config. For this purpose, it is necessary to install [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html). With NVIDIA Container Toolkit installed following Docker compose file will be needed: [compose.simulation.yaml](./demo/compose.simulation.yaml).

Reviewer: ```suggestion
> If you have an NVIDIA GPU, it is worth changing the compose configuration from cpu-config to gpu-config. For this purpose, it is necessary to install [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html). With NVIDIA Container Toolkit installed, modify following Docker compose file by replacing `*cpu-config` with `*gpu-config`: [compose.simulation.yaml](./demo/compose.simulation.yaml).
```
Just to be more specyfic
